ports/stm32/boards/stm32f405_af.csv

```diff
@@ -1,142 +1,142 @@
-PortI,PI8 ,             ,                 ,        ,            ,         ,                      ,                 ,                  ,              ,                     ,               ,                               ,                    ,          ,    ,EVENTOUT,
-PortI,PI9 ,             ,                 ,        ,            ,         ,                      ,                 ,                  ,              ,CAN1_RX              ,               ,                               ,                    ,          ,    ,EVENTOUT,
-PortI,PI10,             ,                 ,        ,            ,         ,                      ,                 ,                  ,              ,                     ,               ,ETH_MII_RX_ER                  ,                    ,          ,    ,EVENTOUT,
-PortI,PI11,             ,                 ,        ,            ,         ,                      ,                 ,                  ,              ,                     ,OTG_HS_ULPI_DIR,                               ,                    ,          ,    ,EVENTOUT,
+Port ,Pin ,AF0          ,AF1              ,AF2     ,AF3         ,AF4      ,AF5              ,AF6              ,AF7            ,AF8           ,AF9               ,AF10               ,AF11                           ,AF12                ,AF13       ,AF14,AF15,ADC                          
+     ,    ,RCC/RTC/SYS  ,TIM1/2           ,TIM3/4/5,TIM8/9/10/11,I2C1/2/3 ,I2S1/2/3/SPI1/2  ,I2S2/3/SPI3      ,I2S3/USART1/2/3,UART4/5/USART6,CAN1/2/TIM12/13/14,USB                ,ETH                            ,FSMC/SDIO/USB       ,DCMI       ,    ,    ,ADC                          
+PortA,PA0 ,WKUP         ,TIM2_CH1/TIM2_ETR,TIM5_CH1,TIM8_ETR    ,         ,                 ,                 ,USART2_CTS     ,UART4_TX      ,                  ,                   ,ETH_MII_CRS                    ,                    ,           ,    ,    ,ADC1_IN0/ADC2_IN0/ADC3_IN0   
+PortA,PA1 ,             ,TIM2_CH2         ,TIM5_CH2,            ,         ,                 ,                 ,USART2_RTS     ,UART4_RX      ,                  ,                   ,ETH_RMII_REF_CLK/ETH_MII_RX_CLK,                    ,           ,    ,    ,ADC1_IN1/ADC2_IN1/ADC3_IN1   
+PortA,PA2 ,             ,TIM2_CH3         ,TIM5_CH3,TIM9_CH1    ,         ,                 ,                 ,USART2_TX      ,              ,                  ,                   ,ETH_MDIO                       ,                    ,           ,    ,    ,ADC1_IN2/ADC2_IN2/ADC3_IN2   
+PortA,PA3 ,             ,TIM2_CH4         ,TIM5_CH4,TIM9_CH2    ,         ,                 ,                 ,USART2_RX      ,              ,                  ,USB_OTG_HS_ULPI_D0 ,ETH_MII_COL                    ,                    ,           ,    ,    ,ADC1_IN3/ADC2_IN3/ADC3_IN3   
```

Comment:
I think the current make-pins.py relies in this being `ADC123_IN3` rather than `ADC1_IN3/ADC2_IN3/ADC3_IN3` because of the logic that detects the channel with the most units.